doc/changelog/04-tactics/11018-lia-in-auto-with-zarith.rst

```diff
@@ -0,0 +1,7 @@
+- The :g:`auto with zarith` tactic and variations (including :tacn:`intuition`)
+  may now call the :tacn:`lia` tactic instead of :tacn:`omega`
+  (when the `Omega` module is loaded);
+  more goals may be automatically solved,
+  less section variables shall be spuriously captured
```

Comment:
-> fewer section variables will be captured spuriously

Reply:
Thanks for the suggestion. I’ve applied it.